scripts/icm-injection-scripts/inject-icm.sh

```diff
@@ -68,4 +68,6 @@ cat ${icm_output_location}
 
 echo "Appending a COPY command to the Containerfile"
 
+echo >> "${CONTAINERFILE}"
 echo "COPY $icm_filename $location_in_container" >> "${CONTAINERFILE}"
+echo >> "${CONTAINERFILE}"
```

Comment:
```suggestion
echo "\nCOPY $icm_filename $location_in_container\n" >> "${CONTAINERFILE}"
```

Reply:
this is more explicit that we want to add newlines only

Reply:
+1, I think it needs to be `echo -e` though

Reply:
that doesn't work

Reply:
ok, nevermind